tests/src/lang/parser/statementLoading.cpp

```diff
@@ -1,5 +1,10 @@
 #include "utils.hpp"
 
+#ifdef __PGI
```

Comment:
What's the warning here?

Reply:
I've split up my commits a bit more to more easily distinguish between the different fixes, so it should be easier to review the Fortran part.

The PGI warning I suppress here is (if I recall correctly, my development server is current undergoing maintenance so I can't check right now) that PGI is not 'fooled' by some of the `(void) statement` calls and complains about unused variable or variable is set but not used.

Reply:
Oh interesting. There's a `ignoreResult` method we use to void compiler errors. Is it coming from there?
https://github.com/libocca/occa/blob/f6333f2c808ddd29399b1f7ea0def461050d9d0c/include/occa/tools/misc.hpp#L11

Reply:
If I don't put the pragma I get

```
"<redacted>/occa/tests/src/lang/parser/statementLoading.cpp
          ", line 468: warning #550-D: variable "statement" was set but never
          used
    statement_t *statement;
                 ^

"<redacted>/occa/tests/src/lang/parser/statementLoading.cpp
          ", line 628: warning #550-D: variable "statement" was set but never
          used
    statement_t *statement;
                 ^
```

Didn't know about the `ignoreResult` function - looks like I can use that instead.

Reply:
Spliced it into the PGI commit instead of the pragma.